### ISSUE TITLE
v1.0 breaks directionalLight() behaviour

### ISSUE THREAD

**Issue by neefrehman** [processing/p5.js#4509]
#### Most appropriate sub-area of p5.js?
- [x] WebGL

#### Details about the bug:
I've just upgraded to p5 v1.0 for some of my sketches, and am running into some inconsistent behaviour with `directionalLight` when trying to add interactivity.

This code snippet doesn't seem to work in a very simple sketch:
```js
const dirY = (mouseY / height - 0.5) * 2;
const dirX = (mouseX / width - 0.5) * 2;
directionalLight(255, 0, 100, -dirX, -dirY, 0.5);
```

It's very similar code to what is shown in the [documentation for directionalLight](https://p5js.org/reference/#/p5/directionalLight) (in which the embedded sketch also doesn't behave as expected). Interestingly with a more complicated sketch the code works as expected and the light source moves position with the mouse.

I've created [a Code Sandbox](https://codesandbox.io/s/currying-cdn-fqpkf) with two sketches, both of which have a version using p5 v1.0 and a version using v0.10.2. In the first sketch, `DirectionalLight` doesn't work as expected with v1.0, but in the second sketch, it does.

I've tried making various simplifications to sketch 2 step by step, to see what might be causing the difference in behaviour, but can't seem to find it.

- p5.js version: 1.0
- Web browser and version: Bug appears on all modern browsers
- Operating System: MacOS
- Steps to reproduce this: [https://codesandbox.io/s/currying-cdn-fqpkf](https://codesandbox.io/s/currying-cdn-fqpkf)

**Comment by akshay-99** [comment]
Hi @neefrehman I think this issue is related to #4391. It was patched with #4406 but since that patch came after 1.0, it is not yet included in the released version. I don't exactly know releases work here but I think it will be included in the next release. 

I have tested your sketch on the patched version and it seems to work as expected.

**Comment by neefrehman** [comment]
Hi @akshay-99, thanks for the answer! looking forward to the patched version launching.

I'm also having another problem where `noFill()` isn't working correctly in WEBGL context with v1.0. I've created [another Code Sandbox](https://codesandbox.io/s/eager-paper-nwtdw?file=/index.html) to show the issue. Do you know if this is a related bug? If not I can open a separate issue for it.

**Comment by akshay-99** [comment]
@neefrehman to me it seems that the `noFill` problem is related to #4350 which was fixed in #4366.

**Comment by neefrehman** [comment]
@akshay-99 thanks so much for the quick responses. Clearly I need to get better at searching for potential duplicate issues. I did check, I promise!

Closing this issue in anticipation of the next release.
